CHANGELOG.md

```diff
@@ -7,28 +7,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
-- Pseudo-transaction models.
+- Option for `Transaction.flags` to be a `List` of `int`s instead of just an `int`
 - Instance method in `Transaction` objects to calculate their hashes locally
 - Additional reliability to `send_reliable_submission` by getting the hash before submitting
-- Option for `Transaction.flags` to be a `List` of `int`s instead of just an `int`
+- Pseudo-transaction models
```

Comment:
just curious, why did you switch these lines?

Reply:
I guess it doesn't really matter, but I was trying to have the changelog align properly with the commit order, so you could just read through easily